Tests/test_PDB.py

```diff
@@ -1577,6 +1577,22 @@ def test_pdb_to_xyzr(self):
         self.assertSequenceEqual(msms_radii, biopy_radii)
 
 
+class FragmentMapperTests(unittest.TestCase):
+    """Tests for FragmentMapper module."""
+    
+    def self_test(self):
```

Comment:
I was puzzled why the TravisCI tests seemed to be passing given we know the fragment files are not going to be present. One of the things with the ``unittest`` framework is by default you have to name the test methods ``test_...`` for them to be called.

Your method is called ``self_test`` and gets ignored :(

There is no reason (other than the history of where this example came from) to call this a self test. How about something like ``test_fragment_mapper``?